From 486db07edc0b1bc8cd73dbf9fa901a7edf99a017 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 22 Oct 2026 13:10:17 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh lasse_collin_pubkey.txt from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 lasse_collin_pubkey.txt | 360 +++++++++++++++++++++++++++++++++++-----
 xz.spec                 |   8 +-
 2 files changed, 328 insertions(+), 40 deletions(-)

diff --git a/lasse_collin_pubkey.txt b/lasse_collin_pubkey.txt
index 4a391c6..1669e2c 100644
--- a/lasse_collin_pubkey.txt
+++ b/lasse_collin_pubkey.txt
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBEzEOZIBEACxg/IuXERlDB48JBWmF4NxNUuuup1IhJAJyFGFSKh3OGAO2Ard
+xsFNBEzEOZIBEACxg/IuXERlDB48JBWmF4NxNUuuup1IhJAJyFGFSKh3OGAO2Ard
 sNuRLjANsFXA7m7P5eTFcG+BoHHuAVYmKnI3PPZtHVLnUt4pGItPczQZ2BE1WpcI
 ayjGTBJeKItX3Npqg9D/odO9WWS1i3FQPVdrLn0YH37/BA66jeMQCRo7g7GLpaNf
 IrvYGsqTbxCwsmA37rpE7oyU4Yrf74HT091WBsRIoq/MelhbxTDMR8eu/dUGZQVc
@@ -11,42 +11,326 @@ jSqR9o05wje8rwNiXXK0xtiJahyNzL97Kn/DgPSqPIi45G+8nxWSPFM5eunBKRl9
 vAnsvwrdPRsR6YR3uMHTuVhQX9/CY891MHkaZJ6wydWtKt3yQwJLYqwo5d4DwnUX
 CduUwSKv+6RmtWI5ZmTQYOcBRcZyGKml9X9Q8iSbm6cnpFXmLrNQwCJN+D3SiYGc
 MtbltZo0ysPMa6Xj5xFaYqWk/BI4iLb2Gs+ByGo/+a0Eq4XYBMOpitNniQARAQAB
-tCdMYXNzZSBDb2xsaW4gPGxhc3NlLmNvbGxpbkB0dWthYW5pLm9yZz6JAlEEEwEK
+zSdMYXNzZSBDb2xsaW4gPGxhc3NlLmNvbGxpbkB0dWthYW5pLm9yZz7CwZEEEwEK
 ADsCGwMCHgECF4AECwkIBwMVCggFFgIDAQAWIQQ2kMJAzlG0Zw0wrRw47nV9aRhG
-IAUCZZwJyQUJGuHiNwAKCRA47nV9aRhGIE4qD/4jdFTe3WPpLgvz/jdlbnSZxr7q
-OS6H/ZJFENHO4SbavXdoXLtj+t6/lqWq890Js8IpWaaiJLowzW1xJMEg99W6k0KD
-3pHUbwPxf0GCSAt/W4JYxdTj+1ggdHjx5yBAmOakjnOH+ZDKQNBnDOI6ghf3ew+H
-9z/b0mQX3rlQbtoqSPZtuDOdFcjCOSwEyqdV+9eNqnv2CoKZkiGoUB1WGCbqKUkY
-KiUJ3WldmPQ5RQYjEi7zZWVac1VuwBA0XOku+W4cCJ5DnPyK7CtMwC84VvaodlOX
-UAK3Y5BIZpZM2Rk6yMX5lFDA5nA8UuHJQRDjTVmh3BIdgRvp0ZV6ogtqNE7RifpW
-aBWDIsCkimcbCJJM+edOLiVZog+ia1Ts8zu33wj7Tnvp5znLc8NLZIqwu1HKLS97
-m+Yf5oC3ObTZtXbVF+OglWe/3ljLHdL2bJxNdtcVlChSNPUW3fgLHk9Fzrlnqdab
-tSGwI/0Ryt00cKjRiMOagTn5Nly6boCtgGYdQafQoSrs3eQjnWVgbNYDMgPyl4k+
-Q5RJLEY7AvtXo7FUEgOTfr9PWmjmc2JzGpxbtwl6sQi6yLrBZTRf1Xao2OjOje6G
-XdUbXNmgOv16sWxcI0s4lX1z28BgHQfwXhBFBRjw2Sy+6TfFXjX24thcpMwvyJ3c
-xhMtdY4N4jyfRjYe8LkCDQRMxDmSARAAv8XAp2PGA/G1KmCrVIzOBm1NPIuqGAYP
-c1l9p0dYdhEgvfw0NXcl5MDv1jbOPZ2PspA8NP7Rqp6LNNXYTeM/eIJDndU5Phyi
-ewFpACAp7Gmm2dL5PUOhu0gIUnQYbN/QdGPoo7bNI646K1Y9aVTBu9fszQssjb6G
-qXHSNM+pskVn9lropO1tLrF0I9VSlSphlCmiQRlzBCZSnxD6UagkPaw1gJnJqnrd
-f9oA6AIavZFdh104fl7y8bMZb6bC0K/5ZD0DLfmYaojkyqRtl3VBu6/ZvXrjsT9A
-QS5x9EdVslUoYY+kUxQm1wi3LIi3mOj6v0IIvgKzjt0X/39E3C42+m8ddTKowFB1
-Y1lEzHiT80YP9a+I+L2bqYgy6Lqs5CxI5qph1xRfg2rY6uvc5rPYk9B1R94jbeKi
-3W8ryHG9QJBNXcd8mCGLM3qylWXTJA4oGITyaIlGCuMeKUfeFNvGijjbEOQ0Cr4J
-CjdACbWJsPEoIOrRFxY+NwJEA39Dkyalyh2l0qTNXTIYhLiDuzl+tWuBX+SjHavj
-9jGyvwr3T37gfzYCNMoZf8GaxAUJMCoGTqnsjTPGMion/DfdNkFDQ+fivdYiVQ9p
-/Njpr38sC83V8dHF/1KkIHImyzMPTdC7l/lMHyC2Gx2dWZOjuOOKit0Qoy3DZoQw
-vN1ZZND9M1UAEQEAAYkCPAQYAQoAJgIbDBYhBDaQwkDOUbRnDTCtHDjudX1pGEYg
-BQJlnAmyBQka4eIgAAoJEDjudX1pGEYguyYQAJo+5SnMMdu+d70mWfUb9PZg7P5C
-GRepHnckx9Sis5oR5s7NNl5j5Yy4J1UwsmrP+mn52ujqewkkVsCq65NGQQx7+tkw
-uKGvnGBkHdrI+aJk86qLMf4DlnNJEmN8t5jTGQfRLbFVf2I8EY6qXAzCSmL9Zs++
-rDUz65GOTB1EP0XmBRsuVYRfDbFezrPQH0JDucbXFi/2BDnl2/Mk9NBoQ0CvB4oG
-tLDiQZ+jV7n1VXXJ1faD9s7i0hOTdcG6rlyIqi/LyAzdCnOYTkmv3U1kdmzkvrh1
-KEiejnM5fj27RE2v191vh3hgZ+X5+uwjNTP0QC4qP8XykQOAA8usOMVZ72lyXCAk
-wiUcRdrAXLN/XbIFNcQ3m4d3W6t60Gk09wFlUKaEltDMlPUsxiSG3qFwFGPBP6UV
-h3mjJMAl1jltLrR7ybez0SczfrcAtdCsKTvgzV9W2TzUfK2R9PBanmXTXK2M7yU3
-IquHt3Je4aSP7XYb5D+ajlbFNvnXOYcai8WryfC5nLAfV4MbPX+UlRaYCqqHVhut
-gK93re1L5mMI3zjG5Ri5jLpUA9toSJCIJIY5zwr/8LL/ZL4TixXlouA17yjkpY/e
-Bjs8cNj1O3aM4jY2FKCS8UbfxOiARk/5kBMRPEZ/mqpMQttzE8KVjOv6fRxy/eVE
-888/gToe5kb8qYwy
-=6rZC
+IAUCaWpSWgUJHqf7jwAKCRA47nV9aRhGIPwPD/9c+3D47ZR6u91BvwZAT1Lp0XvQ
+2pQzh/kf9x4y6uDWwf2oUEEHCEaj9TwtIrswSm0v2ut8jBdxghCfYLZDfs4UUN3d
+rjkDCSaH5tBGTrDn7HrlnCP+7FtcanpH9uRD1L8yYxJVO4M7lvvLAYmYOe2gn6Xd
+rjdFDAQmCLCD5oxNX6w4pbGWDE1mJomtLsUguXuxqsvutddWT/+EK1d9vb1ctxHd
+i4x/xJYJKWTfQ2rhKoUmRbt/LE9LrRtKgxeJ9FKwdzNkjkXdnt11mHs5dO+MQzSZ
+136PEm9Gls64+xZdzaZFbqB7oaNnFsXjAAe3G6Q2p4kn+F/4+i47O4L24DQ+2ug8
+CHcsMyR3217hSP8Fmf+Zklixpn1qAdEMzFXL8FofjEBNpA7yLFA7yZgL5MyvgwLD
+jr4CseXxvXBCav89jm5u5nr7XFTk8hyo08eCnBKKXK+6NpQEG7DmNvd7zqrcYNkb
+cquPxgNMCdEo8gFlogKaj5p4SQ9ux/Q71/2lqYM3b53vM+UZfGE4Ug/3uleLAMJx
+QzlOOdIvU9Uyz253GjaG64bAiaZ4eOrupGS6IyyfbN7ohtDnRkrb/T/RzWscUgqO
+eUzPyGsRx/Cnq5Nqf7sd8rIfArBMmP/vXxrrE/Ksj/H28ywMGP0qTb0XIHYr07zP
+8itwED4lN0kHgwtprMLBkQQTAQoAOwIbAwIeAQIXgAQLCQgHAxUKCAUWAgMBABYh
+BDaQwkDOUbRnDTCtHDjudX1pGEYgBQJnfrhSBQkcxJDAAAoJEDjudX1pGEYgfp0P
+/jrLE1Y+sRCVUV3Zds58F1iNVKFI62cX/ud92nE5uOzLl6qy40YaSeR8mSoO9hdy
++1UvCf5TdYSxwJfd5n+zIz5ukyP6sxR9zMNHbXPpQQb2d9HljH7tM6k99fKWvZbU
+58C2N0y/lTFgqmaZpCiBUf4R4DTMATSBn6fWHQLMBeOZJ406YCFoxxHWSQDuMzhl
+bevui9xS4dJHpfBiXaAINGQdoVFZ481o5RdEef9ZAVC6xRnwWBl2J3DePhTXdqOm
+hB1mideT2UGvG7edLYTNpO4+wR9H1/XFGVafvfexMAnmMI6kMVa+Ysw2jCh0LkVh
+3dIoiforzS49qZxWc+u1x8ZlcAg6AQsCZusNHPe4N8rWy3LimojtZ7/SXSmip/SL
+zremIDRr5rA4xRFOj0H++Mm92HVF/rvZ9VZ+dZF8CeOAiJ8I3icE9UpFxj1eCY4b
+xmpCCk66DsQRHPhbwhlxyPHHiC6NMxeg9RjWz7pz8ImVDeARKV8+TPVRgoIWMS8C
+kw4P31IbwBQm9Bsa88WQgrl65ZKE5UsCYsojdg0YJdSI8DiL0ARieQO0DJ2iFQpT
+9ZyUFYdZp4BkNxwOK+zx0+IXcN1itHR5vzWUHfombXdwmh0FLhoCse+AN4O/Ptou
+vsCfSpeY1Oelzdx1TWi+yQ1eYDZiP4jSiOkmyHqJyLp0wsGRBBMBCgA7AhsDAh4B
+AheABAsJCAcDFQoIBRYCAwEAFiEENpDCQM5RtGcNMK0cOO51fWkYRiAFAmWcCckF
+CRrh4jcACgkQOO51fWkYRiBOKg/+I3RU3t1j6S4L8/43ZW50mca+6jkuh/2SRRDR
+zuEm2r13aFy7Y/rev5alqvPdCbPCKVmmoiS6MM1tcSTBIPfVupNCg96R1G8D8X9B
+gkgLf1uCWMXU4/tYIHR48ecgQJjmpI5zh/mQykDQZwziOoIX93sPh/c/29JkF965
+UG7aKkj2bbgznRXIwjksBMqnVfvXjap79gqCmZIhqFAdVhgm6ilJGColCd1pXZj0
+OUUGIxIu82VlWnNVbsAQNFzpLvluHAieQ5z8iuwrTMAvOFb2qHZTl1ACt2OQSGaW
+TNkZOsjF+ZRQwOZwPFLhyUEQ401ZodwSHYEb6dGVeqILajRO0Yn6VmgVgyLApIpn
+GwiSTPnnTi4lWaIPomtU7PM7t98I+0576ec5y3PDS2SKsLtRyi0ve5vmH+aAtzm0
+2bV21RfjoJVnv95Yyx3S9mycTXbXFZQoUjT1Ft34Cx5PRc65Z6nWm7UhsCP9Ecrd
+NHCo0YjDmoE5+TZcum6ArYBmHUGn0KEq7N3kI51lYGzWAzID8peJPkOUSSxGOwL7
+V6OxVBIDk36/T1po5nNicxqcW7cJerEIusi6wWU0X9V2qNjozo3uhl3VG1zZoDr9
+erFsXCNLOJV9c9vAYB0H8F4QRQUY8Nksvuk3xV419uLYXKTML8id3MYTLXWODeI8
+n0Y2HvDCwZEEEwEKADsCGwMCHgECF4AECwkIBwMVCggFFgIDAQAWIQQ2kMJAzlG0
+Zw0wrRw47nV9aRhGIAUCYzcN9AUJGPjTYgAKCRA47nV9aRhGIIlfD/0cS3q+z1fG
+oSYAR8FX/am39S6v1T0yVyicEFbwNd7x7OCJ6T8qZYb72FjArP8hq/S/1JQOYn7q
+AvzRb+5WLMgxEOUsrqol6jvwXGeyIhA09lSe7uEfv+NehEKs8F/30qIEa8Fwc7D6
+t/WCesDMULFMf4G86XMFVs4gZDS2P54d7rt4+mDn2ZZve2OGcR3k3qr0sgZWNlLO
+k05n15XtBIFIBJbghKXU68ud9I46Ji1Xrnqrn0H/P0jybW3klhp9AzUOfhuoRyUi
+1PLHDsw7d+jRAHGm8NnOPOAqxitYcAyU8SLBcpL3k24kjq6C6UwpStHBzqwQTQIf
+oyCVefxvlYm+8zhtu/CmVXfjB0W+UImCWIeSpj/YzaD3i+YWz8m9lBGuU2WFNO7S
+AdoenJcUdNiwRL7P1E0n8HGM0OsSOks9NOWZHn+Ka36ivsNXAAec6inp0BnDEzdN
+cfrtgFMNBkiEeBD4+8T/HRNtBpYBrlDLtmk2uF86wRhSb+J+pOZeUVIuRQiHg3ri
+dz2U+n6Ld4jqNRS15L+yOhx2yJQ/lyIoAF0caPB3QcubXEYvnBLlZHvpdB5XwYr2
+ROv4OP5bniAvzL4P45jPdrWd/BJUpBD7V3pac83nvcB+z4K91Hnru2bTNatFlmAh
+R+1rN9nK9fggmeudBtgxpnVJAJt5VwF9p8LBkQQTAQoAOwIbAwIeAQIXgAQLCQgH
+AxUKCAUWAgMBABYhBDaQwkDOUbRnDTCtHDjudX1pGEYgBQJgS311BQkXF5HjAAoJ
+EDjudX1pGEYgE0MQALEPq8n56dnB7ciCQmHkm0uwL1sxYJBGU9bURUyyD3oJvlG2
+uifV43WmnVOXMQ0m7X8LsL9USyv1bA4+Dbox6ft81/87bVAz0gFZ3+4HQ+7ffdao
+qH3YRJLnH1JP0Osbf2dUZI+RBAEy6JrvZWsD5ys+fBP+H+7Ug3r+x9/lUMhTbitS
+KpbY2nLvfd0A2u6Ps3LSxxFdU+W2dV2GrA8e3qo2MLW379wKfAeBbUFHVFhcE+kH
+jbP+bb10v2s+sbRg8ZazYFz4QBNQgJ8QTp4i/B9yK54glOMpP1j0OuiAkLYBgWGM
+Ls3Tg4p0IHaEWzgGN7EnCvhfQYk7suEWXTdz3nUCwgQYfCu5xE8Hluja+a9Dwi9m
+fsKa11QnNe/0mXJTqn4wgrWnPtMroaBI5J99Ahdmfp6dQ+2b2YTC6dw5+Zbz84Ng
++zWAJZiDKIUnHyHt4S74zgzEdIUGzszBd/8B7Z9Kn4w/+nX3yoHx8LmhUGcX3mpv
+RCLS5zUP4Zv6CkEX7KYxv6z9UE5HdBaLKbsDMGsnraBSdsL2MwsadJEj7unutpR8
+xOBevn+hCgHDsTElWW1JJGuKbZgf+3gJXjuY7p1H0nfVG2xMUhE2HP7Ttc+wRNy4
+JX92Cp4Lx84Uu30xjXtK5+FvUNwbLi6tkE4iFdylNXu8dWPkhgfmdg5Pj01FwsGR
+BBMBCgA7AhsDAh4BAheABAsJCAcDFQoIBRYCAwEAFiEENpDCQM5RtGcNMK0cOO51
+fWkYRiAFAl5vxdoFCRU2lEgACgkQOO51fWkYRiBpIBAAh9CvUntOvWb26N6fNPUR
+2mssTri/5peiey4UDKoO0oQx1mlCL6WwNrvaIrIvaH3zRHyoUKDd7xblxm/PjfkC
++v+dROe/1oGdQyqpGrLXFFyo9nERzNmi5J6RZOT+6z9ZqwEUMPT6BMwMgFr3VHRO
+q3n0wdtYIri3yXgGo0GKehMz04K8X9vwV8ekOrG0vt6EoOnDDUTH7TRgi1bnnwZA
+ESKC7ZtD7gABI6cZ2AhfnDVDTGExQzbds3RRNJIDbQd88Xk2HhsDHjK1J7V3tBbu
+1LLYyCf+HG9HSgk2oGsZot/HvRiOUEFJG18pYE1FMCthMggjWdSLaLqEf9EO1YWM
+xV7BdKupYW2X1uVXYsf1Ps7DxJtbpjNbW3kCicp4XtLrsXt6SlViTySInpc2SZjU
+B+lCHFODrk3VhF6t3BPen3dnEA47MDIyJsZatjVr3uzl+x3TJD4aCNc7ivT+eVbb
+sgsmiltvd9TVCuv2DggXyHIA6ccTDDrwAssLoeZjA8X2QmPFOO6IYqTgu1gUo2O2
+c2g6lstjXtNDIIpz1pMLJ6DlpjnRUHTgcwtWYBvZ5T08nCFSsLKiirBWFNSDU28Q
+0jkeUZog2HL/JBsRPP2SRI0/7LixrsRgR0FgORHgNUjQdh6eTH57GBhbnyIvuPXX
+6n2Ai3ofV4J55d3ul2naJ9jCwZEEEwEKADsCGwMCHgECF4AECwkIBwMVCggFFgID
+AQAWIQQ2kMJAzlG0Zw0wrRw47nV9aRhGIAUCXERzcAUJEx223gAKCRA47nV9aRhG
+IMgSEACivA8+mQnVbL6mubeAMZpMr5rRc7VanU9kC5IdC0RcMq/hcNfOJR5ncwM+
+JY+QCQVxAHBvHKAuHhC/r1Wl+WB6LYzDwbVszpNdfSnUci+WLETw0BCGhPAQHuUW
+CMYoQEnFSLxqfjshO1CCCfGHhwkhjplG+A5eRZua74cZoe4k05OQCwY1mynDzyw1
+eyYuFZb+TW2YZ3a9/L1UJ7y/BgvxZtq1Zllbt4NhYLZc3V1uzRhtU63+M9gL29PE
+8XM+9jDMrSDrna+3SiwDEJP2SlbtNuxfW8xKtTkmdvwpbV7xBId8NvmABSQX2Tfa
+Ez9yZ1N6U6HzT95v1dsxM2us7ySCkyoQKS9BEfpPpsdFL1VQvfsHdU580eCkHhFx
+PxYxyFfxwlmSeovUlN75e60kF1Lzqagnl6Jd3+pb/hGXd4PBz1Lrl9a1ERuZzKyM
+BG2ih6RRVBE7s/vrLF9+eRkjALjmTj/dcyOaZvvWhmbQRf7d8uZoLp3zkyKsWrk/
+y+u3J1FmshRN4X5tF66Q0ZME/RimidGAreiFZ3iTILNu3xd8oMIPENY/KVX7j/HZ
+FGbOIpKAKqHiJEgX2vub5CVSBuXp/M0a17FJ9l8puy6hH0hlDLplRk4SbzvrALh2
+GK6pOpFJCm6xPto/wOdlL+l/lgwOHqRCDxfB4ErQGEn3qlp3jMLBkQQTAQoAJAIb
+AwIeAQIXgAQLCQgHAxUKCAUWAgMBAAUCV9P57QUJD4Qh2wAhCRA47nV9aRhGIBYh
+BDaQwkDOUbRnDTCtHDjudX1pGEYgtlkQAI6V3tLOXqNnZO81sWjCMfD7G73HYA5G
+8pwK0z1B13J9lJrumv6YWUyzX8HHCxVoqoadIeOpQl1mn0HEehhcF+2FJtwWky6V
+RQPjub52KtaWSljx7yoJdtfqzm/Jz6Wil+dDXgM0fdQeqbeA5C0fV3cA/bM1Lcb7
+6tFWtyIwNPt8JICOpw56I7onBd5qA4Q7WqZe1kDAZrBQ68EG+HIJyg/CTp16Z5s1
+kq14/FXofJFKN2df5Kw1mpLoIdk4ozhHFiqFBlKKq2G1n5KVddeBxPifGBx4OSjU
+MdqHCbeIS9by5PnaGWvjJTKiNj0+03UPOaK247hkNJzuned8pPbCaEXKoqncD0FV
+fd/TchkqvoxIy+2qApyr2Fd4rQQjFw83ElK4dtGRh9uyGz0//vNzuNod7Tx7NwfD
+HLffGw4KI0jGv6Vg6LwkaQvQhkx56vq/kbuxl4n44D8eyKapOdnC2POBlTeU/Kj9
+CPGB9+rO5+x57I+R9UtOfD9YWJ/P9Ga99cdlmkOohQLotpZn2TuzT3hw5SQaqrNR
+Bq9pbZ351F+SPilNXQW/ibtxyoY9JLBpuUtltWV8r1E9R7g67sRE7qsZFk8SQlRN
+WZpN/D0EeI7KIohKgmzw2OZim/gynBDJ/EjWl+UklRo2O4m2erS//MI78A7ASr/3
+DJdvXr9MWIpjwsGRBBMBCgAkAhsDAh4BAheABQkLwe8PBQJUw8HTBAsJCAcDFQoI
+BRYCAwEAACEJEDjudX1pGEYgFiEENpDCQM5RtGcNMK0cOO51fWkYRiCNdxAAkZGy
+yt36yXh8U+RhOFautmh/7R+8GSolgMc/XIW6nWPgasJNE8836cMr/dlhJM2EflUv
+Sa6iP2Clt8vIyzEMaLEV3E3U5JRJCP1LEAxiuEgpMU+BGlocW72TeBVYlYixWqUN
+nMR1sJ1eEXAe7O7q6z6KYCRcxajcMpX2IsxNHFCxQe8zm/YKCH14mZBXWRvwfBIB
+gM54ISwPEdkHJaKmNYZ1wSoxUfSEEv7wEdzyY0eRzMabVzF4r+eTcugikydSlhAR
+RM1HS9beP7Z+E1pIO1MqAis9grA77YhiB2k+80gUEN6rXX5Rq7wInVeEjx7uJr/S
+q42v5mBG+JDsxvJl0/UJzMN3wl8Tei7aLhLg/2GhzmlcZa0WOclJ2mFhmP86ClTH
+VaegEdZVO5+0S1RUj8Tdljw+38+ZlqSXaFXwQ/Dvrp0ClAyqq7zn7Utkuupe+t3Y
+E1kNlNBemDdauaZw5KURB20ibOF/QPNcajNaHBLpUoBxmMEx7pOFC1M47/31PNGd
+n/mN6d1v10/5DvBUK657ELrArVx/ULdO3gMh2tT0PL+KNORZYWciMbITjbISY4EK
+hoooq1W+LN07Ce8UGtaOSTVZKleBhJNmEeuL2Rl4PhcOrEDL/BPW9vKgyRSlKI2l
+jyS3lE/7P7+y6wMXYySYN/YiDyR+ZTPOSIlFicDCwY8EEwECACIFAkzEOZICGwMG
+CwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAACEJEDjudX1pGEYgFiEENpDCQM5RtGcN
+MK0cOO51fWkYRiAg4A/7BXKwoRaXrMbMPOW7vuVF7c2IKB2Yqzn1vLBCwuEHkqY2
+37lDcXY4/5LR+1gcZ3Duw1n/BRSm0FBdvyX/JTWiWNSDUkKAO/0lT2Tg44YLrDT3
+bzwu8dbU9xQt6kH+SCOHvv5Oe4k79l5mro6fF3H1M0bN63x/YoFYojy09D7/JptY
+82oR4f/VdKnfZLJcCViCb0wp8SD2NkDAudKg+K+7PD8HlTWklQQgTZdRXxVZKIJe
+U42aJDqnRbAhJd64YHyClhqut9F5LUmiP5qfLfNhkKDhNOwk2BlrBGBJkSd7wPyz
+cX4Mun/L6YspHjbeVMt9TD7HQlo+OOd2OjAHCx6pqwkXnzeLPEaEcPdQ1SHgrBVi
+AxX3DNPubLP0Knw8XwFu96EuhHZgexE1W7bB4LFsJyXAc5k1PqPDCLsAauxmvI2O
+fI7opG/8wyxDvNgoPjG8fZNAgY0REqPC0JnTXChH31IxUmhNotH8tD3DDTZOHw05
+n5MwwUrEE9xiETVDfFQcMLfxZ9KLz+BC2g1t5LYublRgnCMNJzFgsNUMM02CphAB
+zl/LCLnumr0eyQQ/weV4twEhLwSDmqLYHL0EdYW0Y3CnnU9vmYxQcXKbstS71sEJ
+JYBBmSBbf9GxkOY8BRNtwVwY0kPgxv1WqdVBiAFvfB+pyAsrax9B3UeB7ZSwRD7C
+wXMEEAEKAB0WIQQi1GXytMFzgDsgxt5Z/PIH/qf0RQUCZaFD/QAKCRBZ/PIH/qf0
+RfL5EACutaM2LgYH0WT+GoGQMZY2+ce+gfa9ynC8M+Cw0ue1g/dGifmbDD+0UQqa
+netywxcTb1/sPGnb8PFOU34OHesTfQHSvAKSNeTBtjP2CYGfPQz5Z1NB25XT6pPr
+ijVBUyNjYM792dK8yoXA8KrNMJF0TS/muNE3jQ7IuEtsYiFReCryXM2J5uvT8Cuk
+yCXi9w66JaDpRZx10z7G32J3fJ8yPhWX9GViP5hgBkPYPKGYQDFWnnoanoYd18l8
+OmcTjhpj7bTz/AQ0HfKdpFrq7ALzadIwRhjDIgMMN0LAotmFRXAYry4URqL9x8Zf
+U7x1rDS3v2zCcyU+1J2p40wegdZrfqk2WXpqteIDctR6BQrl+4DxwU1xTKyHIRoj
+KBZaF03+SXG+LFzVmA9KJUpM/byMfuUzZYjL+34v4h101NyM6S9p4st8LCBauHuW
+Z94mE5m3ezjWqTV6xjqf7/xCak1lDtqBZKAw3l55OhOcqMhyYKjxD/40VlgbI8pG
+fV65Hrlj93Fz8AMJHKwduaqWHeWlA3sXyImLJQq7pc5bEqZ2LSBBvCz3znA+fYPk
+gl+lIiWNrFoSdprN9j78tplNm2wG6SU1Emr9HVl6gE7Db6vNSFpKsRh81Jh48W4h
+NKF5AkiKFjYDXmfq248xDCw+gRPD+kftHGu+rlHDFs/IHBmuq8LAcwQQAQgAHRYh
+BGzAUdPB2aSvniYMWNPjlKx0Ti67BQJbeFoDAAoJENPjlKx0Ti67+QUH/2ck09XU
+gXPCIKMz9V1CAPgKRhzpg4BVHs35zg7yqGxD3DQX9V94WqxHGFFCgHo7ll+cvg1T
+4koWKFNkD3Nz3U87oQUqXhGLkZzhsvWsDt9jNlWabsNDM7fhCUZ2Zf/+amOAUJOI
+c8BffoTtWMFAGa6YF7bBR5N1tpLe/VxPOdcXXwRMP24uHzI+gwoUgPxRspPUotw+
+79UD4+EvH8Kk9g/KWMDCJm5UElC9trKwRkut/cc8H4OP8I4roGsh04mmfuxZbiIB
+tCQqEhzqKhPaFdNdYh5B7iq/Xfy3J3SYloYdcHbp2pzLArCXWxTeCjk6Agq9ZVhF
+pOtBiaHOpjxLaojCwHMEEAEIAB0WIQQf5jSO3qgloilw5cSN0wQPEjzD9AUCW3C1
+MQAKCRCN0wQPEjzD9PHYB/9l3Dkg+CmjqkXuIUxDXAge03RF0o0rwFPchbU8FSSB
+hanl3Fjt3F1F7XRNzt6RcAsYvjANZDyZVv4rVfXSpGuecFx5x6ZLWQUZxXeelDIM
+9Jeh7A/lGPzLJB28Z8+F9VVhk7jcuzI586Xv6oBflAeQSI4PmvJGseJZ7b3X76uz
+k4ulreHnnhEEPBIvp3qm6nm933weB7/v6ozvSWU3KjC+ioW3mp0oGcDoB5mCqB3X
+JlgCsTX1nvxwZT+V7B4StXguwIHK40ZIyL/xC7pimes6ikXSNoNH6wocGovaOufb
+kjc5mPv4w1DamLW2RX6eIz10zYbVRMI+JrtrkX4y4aW5wsBzBBABCAAdFiEE9n59
+boAPJA6D68hn/2rJpkZN8JwFAlr5j8YACgkQ/2rJpkZN8Jyl4AgAliH6srzQe2ZD
+ASG3Q2qWNlqtH8ZlvWPIqK9bZKQnySIbIY7XiaJrVygfE59+PwawNH3WaXBAJQ8G
+tYsdrPqZy/zXblm9VJZTJtJxm7sWt0dEdr2UQS5qBPHBNaWhMt7b+4iQ9JhkACN/
+PzN/FCoLAJFuKeMRfV0SlsQ/K+Fd13Sacr+9bTaxVBF21gAsn1o3Cr6w0qqnOaY0
+REnoW6BJQPB270Rg5L0GByvCVC970u5ki19mxqcaFR+uIc/vU1VDg74q3C4yANLY
+M2FrLhEGLHUVsekz11S06SRXiu6q1lOmTwRlQWM4P/maX65bG4FyQXAoQ5n8M2F1
+dsMklF2OLcLDXAQQAQoABgUCVyg+dgAKCRC+7YOXUKUXIYY4IACEJMblHFMvEzQs
+0tUVZoezO1+Raso2bSuNSbWGsVB4UUO7aHiznLkJhJ/YbwN6W71yRwi6F3SpNB12
+mxNgtUBMkShHNOAqAgowCKKaPVV6TwMesr8rhRahBq2iG0sMU0fHmXtURd9uR8MQ
+wCYdFw1Mvq1kReQtGm8WKYls4Esr1nkhJJJTOkaQl4Ywfq9TCusCSaQDpOKbgBx8
+rlTmX6Nq0krsVoCrM1kt7JIWcHeL8hzJgT4oToY3SNV+xv4s8eNfLgwHlzCNkXMC
+610q15EzXlGncmgilF4rnLrtdzHNqciu6wVQLckGLEdhHNBBzCV8gAmUUS/7BML4
+VCDioXAIGuCvMkyE2UVknRAcrDNLwVUTnd+t+m8E0Kq7Zzt5tBy4/EU2fdMXLu0p
+Uqm1aZAp1dSIkV20sfXw00+Ga1PHnrA3GtFB1DPTlwGkTSWaJvW5DeRORKmoGzG6
+wF1VsoI0DoHQFyjXGbC5fCVC0h4gyXA509/fpOIkD/CnAwlGERsgIrMj6IHUrW9p
+2zOF9mJ4kF9WKWbcVM/KvCYyeiMAOZSPxkwGjg654arNip5lbpPs3zTWy75UZiGR
+KGkZ1zPIZ2ZNucvayjTCS9si1y0gY7Q5gK0T82wevzFpRv5Ky98PELraBBmC/NrQ
+o9jDrZZQ64cGqiGAmuDbT28qhlNgx55ttUIAoUXCl+whUDHqzNerA7ccaWkzUYSQ
+XbRlG0A7NYWiQ5GdH4xqjuqV9jYoB/gs7wJgb8H/LMQoyFEsLzdqqNi8Rqli8cR0
+F3+Y3t9YNdfjJ64yU8IDL69931IPUBnmh5fNNQWyUBBv1cTPGVe85cKkcp9EOeBr
+jWLronocCl8oLN/YRHYR55mjnfyS3pP0+q7A8TDWK9Tk5XKrKjPHGHC0s2sZGPNL
+8IVsYO4SupWGig050k3HhNPTy0iYFiLjHmv5Mhemg4aaAXYAy6lqL61TJfOJK+b1
+DOyy9iTTbJlqRRj3n+F4CQ4+kyrzyA+lBbx+DSS+aMkQ59vUOKSdPU2nVWt4bvkL
+Q+vWr8L+xvgg2nc8VZVQEUY3WCAShnhjYWkNaH/2ImzVOUuGn3uiL/VZhZ5ENAjK
+WVLN5jMniTQKYjtoLqf2JHFhy+ETubBNVNa+a+vzjZfBD0UGjZjK5YZGCzNcwqFl
+mPHtohsb7VfMeQhAgSThSui8P0N8AnU7iAMfk8/d56qGcMlTXiZfRaSDkPEatx6f
+MRiLml6J1Z/hXoY0WMUrbaMB8OTjIIHOZD0iBlmSZeRIXeY1lVl1O50ISBO0b882
+4ofBo7w2Bi2WiHLPIG1c3oRKXcx6uN0DsfqmMX1d1la4JeTlZxMm/rOZ9N6XKLW+
+x5ExMnsMwsFcBBABCgAGBQJUtuRsAAoJEJW2GBstM+nsPdkP/2QPf+qYkLaFuD8D
+/9Q7VhsilPIcEfvRRuuvYj5P3QBvsqBCbRTDlKYPxVBZgl/AXOKBB7Alci8SrQ3A
+lVCEXGDUhYuYU2WgRG8LDpQ09+4LZXyCLt+IYR0XHCmi5zfzfYDTDcPMBofcYEPy
+rcWcHG5P22SJQ5CkyZBQUQHpHRBgBQ1m/wGTsvi0MUvGQIdXlpH1dIvKKZivZID1
+7SxNFkrKrWMhrOfM/gRRPTR97RFMCuo0r7bi8EJeFKyAjWyrYXKW/FsB+lF826t0
+DUmFTqX/3KkXQSQOm72/5Lv8cilS9PJRpsQ/b9W9d+0e5mnwwBT07Lx9jLf9FWLo
+bVmr969QZfUJLA8bdg00vMIlg1AXrRqjlXXMO7wa1zxFgzQZsXiV7Iie8yZUfvcK
+yTpukVdvpdSD8bvehGj0vpEzQYoyq4cW+u28MPAUxFk3dMHHSZoFkZDPEiLLaSCq
+ndC9kxWKYdvzbawYUYPgyBDyT6UkxrFDHAhFwmT0PnCHZUCrFqZNLWetg5bxJa+x
+PsIFEkKJUX8eSq6WYn4kts0JYc3V79vAiYbFWy8sOX1AFlCiB/3QqOTSrDS5Ni3O
+Qx/VVpxYMQALVtvdBaniv+LpTKu1bNXUEsMhzNjYt0Kqnmojq4fIG0mOQjCxTWNx
+2EcLk2D2twKG7KMwswY9GXX/+gBXzsFNBEzEOZIBEAC/xcCnY8YD8bUqYKtUjM4G
+bU08i6oYBg9zWX2nR1h2ESC9/DQ1dyXkwO/WNs49nY+ykDw0/tGqnos01dhN4z94
+gkOd1Tk+HKJ7AWkAICnsaabZ0vk9Q6G7SAhSdBhs39B0Y+ijts0jrjorVj1pVMG7
+1+zNCyyNvoapcdI0z6myRWf2Wuik7W0usXQj1VKVKmGUKaJBGXMEJlKfEPpRqCQ9
+rDWAmcmqet1/2gDoAhq9kV2HXTh+XvLxsxlvpsLQr/lkPQMt+ZhqiOTKpG2XdUG7
+r9m9euOxP0BBLnH0R1WyVShhj6RTFCbXCLcsiLeY6Pq/Qgi+ArOO3Rf/f0TcLjb6
+bx11MqjAUHVjWUTMeJPzRg/1r4j4vZupiDLouqzkLEjmqmHXFF+Datjq69zms9iT
+0HVH3iNt4qLdbyvIcb1AkE1dx3yYIYszerKVZdMkDigYhPJoiUYK4x4pR94U28aK
+ONsQ5DQKvgkKN0AJtYmw8Sgg6tEXFj43AkQDf0OTJqXKHaXSpM1dMhiEuIO7OX61
+a4Ff5KMdq+P2MbK/CvdPfuB/NgI0yhl/wZrEBQkwKgZOqeyNM8YyKif8N902QUND
+5+K91iJVD2n82OmvfywLzdXx0cX/UqQgcibLMw9N0LuX+UwfILYbHZ1Zk6O444qK
+3RCjLcNmhDC83Vlk0P0zVQARAQABwsF8BBgBCgAmAhsMFiEENpDCQM5RtGcNMK0c
+OO51fWkYRiAFAmlqUj4FCR6n+5QACgkQOO51fWkYRiDGSA/7B/0qTpCC3X6aq6Z4
+R3WIDYusIzbDVQbXYtGIzR+xMllPEUE63jonbdhoPnBijJ1oakT85LhWJedlMju+
+d2lOvYoAjtlqI2/4mAMDnHeiU9V0wL07QN1UINeQs7HIl7OxLV2J0Z26OawP1dff
+gSi7QAKCsmdCphcis36v9K7G9G7tiHHYz7N1fNyHk+RggPY9dFRj43PPfRWnHVgn
+k9R6COIHmos2+V0zuQ5vXZw49ViWvf+yHnN8BwE1dxgUCvvxu5hknvxfo/JCMhyx
+c95mgq3iH9VHxQClhyaN92S6zoKWStmoD+0vgKRqdcYLCmZ2Po/8GrLeq7WfQZ8h
+Tv7AGdvdzs7yqleDjU+c1dmX0c5cKzbmVp/1czS65KX5OZ2wlrngPFxQQy83gLhG
+4qEhFdVHMqWMYJFkL75eBDj/AzrEW59u7XxRwFpnCbqW45/W8OoyixC1JLG3+0It
+QV3Wl+aKXOJ+iynAUNU59o7G2Q3y2UkBIBHVOaYvkAGYcO5xsy0Wku1554pSNeU5
+e+JiqthJMNckvZPV13U2T/PmMA5o0dOz79PzBCzziHtZPto9gICdHngTEGmwn3Xr
+RWB8pTJBBGJsjHycJjwSBpPxzQv8vss5aU24Kt0wzRMj59HH/YsjBUERhdC3WszN
+4yZ4lNKcZEbHf3GOxjlq0rDSijzCwXwEGAEKACYCGwwWIQQ2kMJAzlG0Zw0wrRw4
+7nV9aRhGIAUCZ364RwUJHMSQtQAKCRA47nV9aRhGII2iEACMbNrtKDaiohSufHf5
+aUoPrFoMDt1hvXAoYULz5yXcgHVypZ8PP0kspKrbjL9fzdvZmEjuyt7AiEr6Ak0d
+iqk+eOqPgtvwqkrN1hLl9UqT0BlT1C4k8Sy7GYdFoSaynIZldzUQAj8aLnoqrRaL
+CTwOrtbH9opTfPQKxsc7XiLk6clMua/fBh1CubL41YeLM/ir0zZRhRzd5wKEewYY
+g3+kYENEN7pJBiar7WElFd0blZIEfuxRwxbG+kUZspHJvmErc9z9GEzCY2y2HsGk
+C8ymZy1p0jdfDUayE8BFInAV5HDhYxdfHe412LAM81+5dvCxYucoFrjjr0+bOxM0
+5lrcufqq3hx54y+EgkGNq5G/QIqVE6qaA4Qc/dUIr03UPxLCZT+ntPIcGmu4Xmam
+VlstXka/ERMw9q9xn0NhHoD5MLInYrwwZSuD4Fp5RJdOkWxNXV6Gpl3zydatEhZZ
+MN8zFvm6mD9Y08ayVQJVxX/Kk93eaV8/O9UdTTz/3cjyZ4vOOAYuNqvCRyGWilme
+kELD9tExjAa72yPKjAjNYB+fL3AVgR7aZtpBhI1XScpe+UYIwn9VR6j2m+gNP/rQ
+ARpS3+a5vZMTpm9sAwlvMT56PwPKbFVnGBO4BEU+gXam5K90mcPdosxggOJteztT
+D3+r4/54G0UTr7hCNdRyzpgSb8LBfAQYAQoAJgIbDBYhBDaQwkDOUbRnDTCtHDju
+dX1pGEYgBQJlnAmyBQka4eIgAAoJEDjudX1pGEYguyYQAJo+5SnMMdu+d70mWfUb
+9PZg7P5CGRepHnckx9Sis5oR5s7NNl5j5Yy4J1UwsmrP+mn52ujqewkkVsCq65NG
+QQx7+tkwuKGvnGBkHdrI+aJk86qLMf4DlnNJEmN8t5jTGQfRLbFVf2I8EY6qXAzC
+SmL9Zs++rDUz65GOTB1EP0XmBRsuVYRfDbFezrPQH0JDucbXFi/2BDnl2/Mk9NBo
+Q0CvB4oGtLDiQZ+jV7n1VXXJ1faD9s7i0hOTdcG6rlyIqi/LyAzdCnOYTkmv3U1k
+dmzkvrh1KEiejnM5fj27RE2v191vh3hgZ+X5+uwjNTP0QC4qP8XykQOAA8usOMVZ
+72lyXCAkwiUcRdrAXLN/XbIFNcQ3m4d3W6t60Gk09wFlUKaEltDMlPUsxiSG3qFw
+FGPBP6UVh3mjJMAl1jltLrR7ybez0SczfrcAtdCsKTvgzV9W2TzUfK2R9PBanmXT
+XK2M7yU3IquHt3Je4aSP7XYb5D+ajlbFNvnXOYcai8WryfC5nLAfV4MbPX+UlRaY
+CqqHVhutgK93re1L5mMI3zjG5Ri5jLpUA9toSJCIJIY5zwr/8LL/ZL4TixXlouA1
+7yjkpY/eBjs8cNj1O3aM4jY2FKCS8UbfxOiARk/5kBMRPEZ/mqpMQttzE8KVjOv6
+fRxy/eVE888/gToe5kb8qYwywsF7BBgBCgAmAhsMFiEENpDCQM5RtGcNMK0cOO51
+fWkYRiAFAmM3DdkFCRj400cACgkQOO51fWkYRiDWZw/4h4KT3QgVndItf6yJplAJ
+AjNwP4vdT6vC6Iw8ZzEF+3kMFZ61l72Wawf1DgkePQHjCXwIjMvlT+gJz4nbCJmp
+YEXvDruiMzpGu64nJE3GhbKyQOIJJi1ygyKzwSraQFia7Pgd6LgxgFNfRH8cXd0n
+M6181gaiUu1ri9fMy6hsFq2xam9PDRTrSQc2LEpHDfDrW8XKFTxpmRNIfooJGG2m
+TLDnQYwqhOfhQekgBkn2awWqSuXYvvdEQNY9LXF1L1MD+HwmNEcfcGa5j3NUdg/C
+R6wUM315qHeua3dVUjqvQfAFmcNZ+p8A3O/El2gk/5vkqJjg5rJAjknP6urO01G9
+rSsLL87LfaRKjsxJ/lu8MDlsXMjisWOAFeTnyDLwc0DtsespIfm5IVI+eyKL9m+6
+9rVPawFXNXi540IDzfvLvOtP3UHXzLmuVSAqhQjepS6sk+Mx7dPEtba2wccs12R/
+Gqo404LsHv6uWqzgX8bN7WkG/zjxbhl6fZoIglUCxnLQ7dv/nTXyzp5lqHlMtqQa
+ktd9NrAQfp36xhUxZiQuMqc2PLkBRvfHcQaM6jBPN+iqzIYgW3iyIIV4LDkBx7fo
+F8kFc787JHnVMWeJsc2dQ//iXyYcMRr8WRZ+bABi2wJkW16CL9Hbh5PyVthdb7f0
+tN683nPMt+wdyy1pyDvSysLBfAQYAQoAJgIbDBYhBDaQwkDOUbRnDTCtHDjudX1p
+GEYgBQJgS31gBQkXF5HOAAoJEDjudX1pGEYgwu0P/0e4ozimeAiZy7NjDNCZ2/iP
+bphjKHiNWwoSZVZOJFx6ESBQiWtaQK7erN3k0r5F61LuQnww+fMRR+Nhul0LrKsX
+qfWZKtlnhUkyRXZ6/ftsiBcz5anWYIAZuM3FCeOf1FptP+CMiqYa5GcA/tGxJ45K
+47+A72HY+15yLPbe6yxOKUH7xxOihARBBl7oq//O6S8v5xxJ6EsexnupV9FQCa23
+ycWRdcT6zyN8t+Gqy1ojb9Em7nCK1o9xczwyfPYT3loBIBtnLR5Ci33Q+9/Tuf3K
+4Le255O/O+VfHeHlTfJPji0g6bMA0hCNrLVMZ2b5EEnZljKHItrCVnY1VRddKnhB
+llc8DRRZsX6lvtD1x0oM0VW68YGWO55rRh3RPaj6JsOrjcfOJf2WX6VJeT2aq9bV
+RwM5rFatKybUZzU72DfCofnEcCG1jwY+H/tWABrCyQ+SaeWQxbqlg/LOJtt4hIkv
+WB3WMhPrfLpqhWu02ij7BgmbbzRE5+WHj7lA6jpAn6ObvR+RdIb+onlrz+oI9MeQ
+lz+umQvr9MNAAlRGL1GEMALSBvjQe26xs3UtkQD6LRxZOZhdqn4MHhhHikCmKWlo
+bzsz5VSiRHjGmfHu9NvYw9rsx16e+L0UQacpdp2ZPzTfy+V/PPkYZRMyVWKf0FA9
+Ol0D4+lGIm8omBUN4AU6wsF8BBgBCgAmAhsMFiEENpDCQM5RtGcNMK0cOO51fWkY
+RiAFAl5vxcMFCRU2lDEACgkQOO51fWkYRiAETg//Z/wItCweI0pEWqyz6mRc2VbH
+bbSr9P824A1QsQ0ZAeyfUVeA88Zv4kTlDaT++Dwpdb3b6ct4SVBlIVqRhT2IgrPT
+ooGTvm+wyuu/Z8pXYH4FRi6ItifZd/Z4IH+yp6MCBhP/PpwTNod54+kRGTvItwcN
+9zCt1EaYk3+p3i7BIMuOd6vJLj7B0GObyS+X372aalsmq/FUEWi66nysu4NsX+jf
+f3Mb+MFUux8Int2XJlTTOJtkmh0upSSqtnNHKgUPSsOkSmyQ2HXUbugubWgoWUwd
+8a5SCte8TZE27lqeBNHAZ1EVH2uCel3L2PPvpmwSWp3pu4Mu70AOx3CtwwXSqyxv
+IuEHNTewSiUbzPeMsY0aTb2vnGkX5XsDqPGqFnKdwCYOIwFt8vkUBnyQ8Vct67hh
+0F6CGB8WIuIupS2ySt5sPb3tVbMWmaA4Dwl2NwkeHCOVCWxpmc2WRlRK+Dpw2tNL
+WMwRdAqkpiuLgWRHvrpYMKIwALpABkEilOqPBgG4RB3zsCzLAKU89o6xLaTZ+liD
+rExvoovLBvUeBwkM9+sFNKcCmbQ7I4OHR6vq0wRscWCEO6aKoQoDhe8mj/JgWFjZ
+c6N7i7CV1fWmeRlqjsays4ZinDPQ2yXo4OZUC+msu/RsE17yuhPsOCA6F/hzXHY7
+KgS6FMyLR+dodsjX0GfCwXwEGAEKACYCGwwWIQQ2kMJAzlG0Zw0wrRw47nV9aRhG
+IAUCXERzXAUJEx22ygAKCRA47nV9aRhGIDqVD/46sXUGfW5A2dP5vk9d0zTERwUA
+vgzZfZJWTJ38AERiqCbFLonVbqMF4Yj2rCat50nSVvI8UnHO61qTSWB/nwdCjTgm
+Hl4N/hhplWSnY/+OcMOgHJ7MF3w7aBvCZqgVN6h/2w2oUCI18KHF/KkoWu66DrqW
+hOzWP0feI3UCgLuzZP7KJ6oE6yv3w0I8vV/2G4Mm7HSgstLur5vZyO/MyiV/x2OR
+33H25HhwHEzZMm0vO+EAR4FWcLqX/70rv5Qy4QY0aLSC5EvY3X9Q4P0QxiEjmRsG
+gm7dh03Pxbr01JH5sIW6gnrCs0oxmdnLt8XyMYkvGdUdllVUe1XX0UT6buHetWNO
+v6RoS9g0E+GEI7I7qEl7x9z7rB3AWwOU6FFteggBFfXI/AmRIfBg/NUdM4Co1sIj
+yyyQcGgIYiq9MvyGRSey9/td9yaQpB02oITfyqwShRY3a2CnXr6lnW4uwa0LrNA6
+eBDVub0GLADvJiqwagt8uJqSBq8aGQgn9xhPUptKJlwKfKYHVdVSn95tAusFKQ9E
+CgW3Tteu76pmwBhgtieWqcW+fzI04+nDD2xSozlEaEoaDHD4Ti70wW3VWzUd2E6H
+DlWw+uG7Ll9E/O7fCsZ2obEIUWRjzQKb1992CcfUb/kuwF2CtAVVaGKSZLbWRS47
+D8RFJS+CAn6a3TqNLcLBfAQYAQoADwIbDAUCV9P5zgUJD4QhvAAhCRA47nV9aRhG
+IBYhBDaQwkDOUbRnDTCtHDjudX1pGEYg398P/AiAbnJg7FzG8xliT7qUU6/Wnh38
+TnJDVvPcdBfONtSRVhQCleHAdQ6/HVXToGdm2n3HdvLkKeGNZk2xYwiJIEeEtdKW
+LQ4HEV7SccYFkQB69dT5xDJ50WYpqYIEnJnn4QqPrB60U8CIjxnb0e77hO5MINno
+aqHv9Hw2wNO4j9F8WVVkpEiS53tcB3thed/P/0oaGMUZmCgZZijS196YcUwBOv5R
+Wsf51+4wZEZcdzP5kzG2sY/P+kDOB8+wpRI7mOJTVd6KFGohJp/PpC6HrMIBiPwx
+LGDFEvC89kA5BN0w/n0psmoQVsKrSj4yebE6z0WmWdPE4ATbJS+cf3CH2lGs8JLK
+6+yuwLzgy4eDUiCnR6FvLed+YmC2o7ADFChSFq3CDmGgo4+AsBGITp5YCbJwiU14
+u73KLftueN3ABzNX4pe35amXKimbp4xqOF52H/+BiJpiHPHYVIjmoylf7O/WaYzs
+GAWCu0aSBL1EoYEG5DSMkZkDiTKq06hWru85ujkXt8VgWyzhzU/NcobbZD4vABNx
+fNVBo8bxkyBK+rZR8gCwSkz0jut4VC62nbn7QnABz9hAa0ep5GQkQ9okqMH5HbiF
+UmfcYeMbhqGUOpdfRvcF+m+JSVBAeTsinqCreI8NA0RTv+b7v1FNdxJXqHi0RhPF
+bsGkJOiFW3aB5/aHwsF8BBgBCgAPAhsMBQJUw8GWBQkLwe8EACEJEDjudX1pGEYg
+FiEENpDCQM5RtGcNMK0cOO51fWkYRiD3mBAAiUh7VKSPHfmyv2jMEKVF/HbsFxdf
+kz4PnT4rzOpN06K2PRO4AfZiDPPPKMCJerX0f8kGRblFwhbPLl9nWnGzdHwMLoLA
+RzM8ZhU8Nkao6UuWymOvXksPt7xSE4r7pXAmdOqmXPNblxgjTFUzJFI9Q85bKhhc
+8L6VXE//fTEr69MBNd0rP0q/jvFUN/HMQGVPHzbAfC4pm5OutEnq6sV3WDCxVU4l
+WEtJCoOoFPeu7r1YGp6PdEmaDtHgFghHaItoUp744FON23YGr79/yMz2rV/Nvx0E
+1YgkqAjy9UqnD944eIeuH0S7Zh1k8VWlNq1LmvDtbONNQgWnG0QlysKA0MTKIccd
+gv2Io3RqKbunlrVApLD2dO6PqtAZ7fE+hSoGeGRAhF13cMI3wVhVwL9ePzas359q
+IkTp33oi3Nwb0bTNAYgQslaUZmPeSFiw4DT1LriCowWTn9GDePBYe7EcAppOnRlP
+k/YxWDB2HDBLAGIXwrqgvHum2Ipe9//Zp4zr6mRXmELaPXegDrcpUg6qx/F0qIsq
+LKUBhn3LNtTtivTY99g2wXYFBUvVpH+15MiW5xBqL3+w4jitTyRvA0DdOAm5KdlZ
+UH7iOxWHPwP6lWB0TonWoXu1QBkPWqPkAELTUwsNzyxGzkie29l0ob7rdye9b+3A
+I0IYf3NMxVBM4x/CwXYEGAECAAkFAkzEOZICGwwAIQkQOO51fWkYRiAWIQQ2kMJA
+zlG0Zw0wrRw47nV9aRhGICaLD/wOlfPc3F9QB6qeXbSl0WvZgk77bwPsFOjOG8v4
+EuxFKLOhh9tqnumNYhI6k3gYB5Jg9tkxT4x8n1PZw0DrN7N1PimRNbK4yM7x1aK9
+WpyIZfNiED0cc+++SH9U0+vK3ZlGnY3PWOl3tofH7yIa5JF6UM/z0y1voKiY38bL
+Tk+FlIBqTa2EX9k9wN0YUViwVWpF385UINWZ16f20H2jEG64HrmQ+W1xfPI6KFGN
+7tVS2mlsK/E8wDQQ2Rmx9/rs47LkmPyA7Kc3aPitLjQKF0h6MAGJ5QYPGhrm0zwb
+yXWeWBOoHaNfvkpOZCc9UtCTWJ81fwsIfp3vb22v0R3Fz0qhIIJvQb9ZON3gw2kj
+uOGMu51IXfl++yzmZrFsEQsFMatOYBwsWlE6jwafKSsrJ9vyVSOYpNmg6aCywVOY
+MgecMK3rgl5u6qBxmgtoYAYqS4B7gQyx2Ujp/eU1MotWQOv/qdVVh0rSV5Cx8Wai
+G8+OgymvFL8vNR59d3KnW01k0mI4xKuCXdADEp3sF9pzGf+HTd8YG93bN+tXEMlW
+heyc8gM1DoskZJ8Oaxob+ZGBkkS6dUsZAV7aexWo2ZDGm0tpPO3LVm/Z0I4Sblb+
+lJ6QsIs94MroqZfxlVFos+Ph11EIAZkxqL5ubSf/SyMD3cNsG1LRfTCT6Qi6k8Dk
+pZ0rkw==
+=Vz6f
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/xz.spec b/xz.spec
index 6f750bb..894c5b9 100644
--- a/xz.spec
+++ b/xz.spec
@@ -5,7 +5,7 @@ Summary:	LZMA compression utilities
 Name:		xz
 Epoch:		1
 Version:	5.8.4
-Release:	1%{?dist}
+Release:	2%{?dist}
 
 # liblzma - 0BSD
 # xz{,dec}, lzma{dec,info} - 0BSD
@@ -37,6 +37,7 @@ BuildRequires:	gcc
 BuildRequires:	gnupg2
 BuildRequires:	perl-interpreter
 BuildRequires:	autoconf automake libtool gettext-devel
+BuildRequires:	openpgpverify
 
 
 %description
@@ -91,7 +92,7 @@ commands that deal with the older LZMA format.
 
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -p1
 autoreconf -fi
 
@@ -184,6 +185,9 @@ LD_LIBRARY_PATH=$PWD/src/liblzma/.libs make check
 
 
 %changelog
+* Mon Oct 12 2026 Jakub Jelen <jjelen@redhat.com> - 1:5.8.4-2
+- Verify upstream source signatures with openpgpverify
+
 * Thu Sep 10 2026 Richard W.M. Jones <rjones@redhat.com> - 1:5.8.4-1
 - Update to 5.8.4 (RHBZ#2531115)
 
-- 
2.55.0

